src/javascript/app_2/App/Middlewares/is_virtual_account.js

```diff
@@ -0,0 +1,3 @@
+import Client from '_common/base/client_base';
+
+export const isVirtualAccount = () => !Client.isLoggedIn() || Client.get('is_virtual');
```

Comment:
I still suggest using a more general name for both filename and function as:
- It's not only about _"virtual account"_
- We know that the condition might change repeatedly in the future.

Reply:
Yeah I was thinking something like, `isClientAllowedToVisit`